planner/src/main/codegen/includes/model.ftl

```diff
@@ -42,6 +42,25 @@ SqlNode SqlPredictModel() :
     )
 }
 
+// EXPORT MODEL "name" WITH ()
+SqlNode SqlExportModel() :
+{
+    final Span s;
+    final SqlModelIdentifier modelName;
+    final SqlKwargs kwargs;
+
+}
+{
+    <EXPORT> { s = span(); }
+    modelName = ModelIdentifier()
+    (
```

Comment:
I think you do not need those "(" and ")" here

Reply:
got it, removed